### PR TITLE
fix: Filter out control plane endpoint from Nutanix node IPs

### PR DESCRIPTION
**Pull request by jimmidyson**
**What problem does this PR solve?**:

**Which issue(s) this PR fixes**:
Fixes #

**How Has This Been Tested?**:

**Special notes for your reviewer**:

### CODE REVIEW

File: charts/cluster-api-runtime-extensions-nutanix/templates/ccm/nutanix/manifests/helm-addon-installation.yaml [hunk 1]
```diff
@@ -16,6 +16,8 @@ data:
     prismCentralAdditionalTrustBundle: {{ `{{ printf "%q" . }}` }}
     {{ `{{- end }}` }}
 
+    ignoredNodeIPs: [ {{ printf "%q" .ControlPlaneEndpoint }} ]
```
Comment (mhrabovcin) [inline charts/cluster-api-runtime-extensions-nutanix/templates/ccm/nutanix/manifests/helm-addon-installation.yaml:19]: I am only guessing, but isnt the endpoint value including the port `x.x.x.x:9443` ?

Comment (mhrabovcin) [reply charts/cluster-api-runtime-extensions-nutanix/templates/ccm/nutanix/manifests/helm-addon-installation.yaml:19]: Its not, its the host part only...
```
ControlPlaneEndpoint:              nutanixConfig.ControlPlaneEndpoint.Host,
```

Maybe use a different name for the value?

Comment (jimmidyson) [reply charts/cluster-api-runtime-extensions-nutanix/templates/ccm/nutanix/manifests/helm-addon-installation.yaml]: Changed the name.

**Review by fatz — APPROVED** [review]

**Review by yannickstruyf3 — APPROVED** [review]

**Review by mhrabovcin — APPROVED** [review]

**Review by thunderboltsid — APPROVED** [review]
🚀

**Review by faiq — APPROVED** [review]

**Pull request merged**